apps/portal/components/Skeleton.tsx

```diff
@@ -43,35 +44,57 @@ const pages = [
     name: 'resume',
     svg: CameraIcon,
   },
+  {
+    uri: '/',
+    name: 'sign out',
+    svg: HomeIcon,
+  },
 ];
 
 const Skeleton = ({ children }: any) => {
   const mobile = useMediaQuery({ maxWidth: 900 });
   const router = useRouter();
-
+  const { data: session } = useSession();
   // if (disable) return children;
+  if (!session)
+    return (
+      <>
+        <Background splotches={3} />
+        <div className="h-screen w-screen overflow-x-hidden flex">
+          <div className="w-full relative">{children}</div>
+        </div>
+      </>
+    );
 
   return (
     <>
       <Background splotches={3} />
       <div className="h-screen w-screen overflow-x-hidden flex">
         {!mobile && (
-          <ACMDesktopNavbar>
-            <Link href="/" passHref>
-              <ACMDesktopNavbarItem isLogo>
-                <Image src={WhiteACMLogo} alt="ACM Logo" />
-              </ACMDesktopNavbarItem>
-            </Link>
-            {pages.map((page, idx) => {
-              return (
-                <Link key={idx} href={page.uri} passHref>
-                  <ACMDesktopNavbarItem $active={router.pathname === page.uri ? true : false}>
-                    {page.name}
-                  </ACMDesktopNavbarItem>
-                </Link>
-              );
-            })}
-          </ACMDesktopNavbar>
+          <>
+            <ACMDesktopNavbar>
+              <Link href="/" passHref>
+                <ACMDesktopNavbarItem isLogo>
+                  <Image src={WhiteACMLogo} alt="ACM Logo" />
+                </ACMDesktopNavbarItem>
+              </Link>
+              {pages.map((page, idx) => {
+                if (page.name === 'sign out')
+                  return (
+                    <ACMDesktopNavbarItem key={idx} onClick={() => signOut()}>
+                      {page.name}
```

Comment:
Can you try make this a `span` so that we can add `cursor-pointer` to this as well?